lib/deck.rb

```diff
@@ -0,0 +1,27 @@
+require './lib/card'
+
+class Deck
+  attr_reader :cards, :count
+  def initialize(cards)
+    @cards = cards
+    @count = @cards.length
+  end
+  #
+  # def cards()
+  #   @cards
+  # end
+  #
+  # def count()
+  #   @count
+  # end
+
+  def cards_in_category(category)
+    cards_in_cat = []
+    @cards.each do |card|
+      if card.category == category
+        cards_in_cat << card
+      end
+    end
+    cards_in_cat
+  end
```

Comment:
tomorrow we'll talk about some other enumerable options. This is a great example of when an enum like `find_all` would come in handy.